python/tank/deploy/tank_commands/copy_apps.py

```diff
@@ -50,7 +50,11 @@ def __init__(self):
 
     def run_noninteractive(self, log, parameters):
         """
-        API accessor
+        Tank command API accessor. 
```

Comment:
it was a big search and replace. At least it is a little bit better now than it was before.

This is part of the toolkit internals, so i don't think it will never reach the sphinx level docs! see here for tank cmd docs: https://support.shotgunsoftware.com/entries/95441117#Executing%20tank%20commands%20via%20the%20API